java-kotlin-aot/README.md

```diff
@@ -12,44 +12,27 @@ This repository contains the code for a demo application for [GraalVM](graalvm.o
 This is a simple Java/Kotlin application, where a Java method accesses a String from Kotlin and calls a Kotlin function, which later accesses a String from a Java class.
 This example demonstrates how easy it is to interop between Java and Kotlin.
 
-1. [Download GraalVM](https://www.graalvm.org/downloads/), unzip the archive, export the GraalVM home directory as the `$JAVA_HOME` and add `$JAVA_HOME/bin` to the `PATH` environment variable.
```

Comment:
Remove "GraalVM Demos: " from titles:

`# Java Kotlin Interoperability and AOT Compilation Demo`